index.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="./css/style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <!-- hero section -->
+    <div id="home" class="home">
+      <div class="home-top">
+        <div class="top-content">
+          <img src="./assets/facebook.png" alt="facebook" />
+          <img src="./assets/twitter.png" alt="twitter" />
+          <span>Logout</span>
+          <span>MyPage</span>
+          <span>Korean</span>
+        </div>
+      </div>
+      <div class="toolBar">
+        <a href="./index.html"
+          ><img class="logo" src="./assets/medical-logo.png" alt="logo"
+        /></a>
+        <div class="toogle">
+          <img
+            class="mobile-menu"
+            src="./assets/mobile_menu.png"
+            alt="mobile menu"
+          />
+        </div>
```

Comment:
- [ ] Great job working on the hero section but I think it would be great if you a;so keep the color as the given design since in your it is blue so please change it to be the same 💯📌

- Your design
![image](https://user-images.githubusercontent.com/58553711/197082115-794424d4-02a5-43e2-aecf-0669201cad9a.png)

- Given design
![image](https://user-images.githubusercontent.com/58553711/197082182-e3cbfd98-21d2-469e-8455-f7058ef39ad1.png)

Reply:
Thank you  @Reem-lab  for the feedback. I understand that the color of the hero section should match the given design. I will make sure to make the necessary changes to ensure that the color matches the design.